SwiftUIProject/SwiftUIProject/ContentView.swift

```diff
@@ -6,18 +6,99 @@
 
 import SwiftUI
 
+/// Button with Alert and ActionSheet
 struct ContentView: View {
+    
+    private enum Constants {
+        static let exampleAlertText = "Example Alert"
+        static let showButtonText = "Show"
+        static let spacing: CGFloat = 10
+        static let exampleText = "Text!"
+        static let changeColorText = "Change color"
+        static let alertTwoText = "Alert two"
+        static let alertTwoMessageText = "Do you want to change a color?"
+        static let yesText = "Yes"
+        static let exampleActionSheetText = "Example ActionSheet"
+        static let changeWordText = "Change word"
+        static let badWordText = "Death"
+        static let goodWordText = "Life"
+        static let questionWordText = "Which word do yo want to see?"
+    }
+    
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+        VStack(spacing: Constants.spacing) {
+            taskOne()
+            Text(changeText).foregroundColor(textColor)
+            taskTwo()
+            taskThree()
+            Text(exampleWord)
+            taskFour()
+        }
+    }
+    
+    @State private var isShowAlertOne = false
+    @State private var isShowAlertTwo = false
+    @State private var isActionSheetOne = false
+    @State private var isActionSheetTwo = false
+    @State private var textColor = Color(.blue)
+    @State private var changeText = Constants.exampleText
+    @State private var exampleWord = Constants.badWordText
+    
+    private func taskOne() -> some View {
```

Comment:
```suggestion
    private var taskOne: some View {
```
  если у метода нет параметров, то делаем из него вычисляемое свойство